atmo/clusters/forms.py

```diff
@@ -10,18 +10,12 @@
 from ..keys.models import SSHKey
 
 
-class SSHKeyChoiceField(forms.ModelChoiceField):
-
-    def label_from_instance(self, obj):
-        return u'%s [%s]' % (obj, obj.fingerprint)
-
-
 class NewClusterForm(FormControlFormMixin, CreatedByModelFormMixin,
                      forms.ModelForm):
     prefix = 'new'
 
     identifier = forms.RegexField(
-        label='Cluster identifier',
+        label='Identifier',
         required=True,
         regex=r'^[\w-]{1,100}$',
         widget=forms.TextInput(attrs={
```

Comment:
should we make the Spark cluster identifier help text match the job identifier help text?